.github/workflows/ci.yml

```diff
@@ -98,12 +98,33 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18.13.0
-      - name: 📦 Install Dependencies
+      - name: 🔍 Check Dependencies
         run: yarn install --immutable --immutable-cache --check-cache
 
+  gql-codegen:
```

Comment:
Here is the Terraform PR to make this a required check https://github.com/CruGlobal/cru-terraform/pull/5838